app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -302,6 +302,9 @@ def pod_definition
   end
 
   def inspector_image
-    'docker.io/openshift/image-inspector:v1.0.z'
+    fq_inspector_image = @ems_configs.fetch_path(:ems_kubernetes, :image_inspector)
+    return fq_inspector_image unless fq_inspector_image.blank?
+    _log.warn('image_inspector not explicitly configured.')
```

Comment:
I think that when user will read this warning in log, it will not be very clear what is the problem and how the system behaves (for example, from the log it won't be clear that a line after this there is some hardcoded default value returned and used)